### DISCUSSION TITLE
Reactivity or mdeia-queries

### DISCUSSION THREAD

**Discussion by Seanmclem** [prevwong/craft.js#210] [Q&A] [answered]
Is there any way to build into a page made with craft.js -the ability to respond to window-size or size changes?

**Comment by ankri** [comment, marked as answer]
Can you maybe add some more context?

I have created a [codesandbox](https://codesandbox.io/s/craft-basic-example-media-query-n1nwm?file=/pages/index.js) which uses media queries. Did you have something like this in mind?

`index.js`
![image](https://user-images.githubusercontent.com/2842920/110843306-69700e00-82a8-11eb-8a21-b844a3507e86.png)

`styles.css`
![image](https://user-images.githubusercontent.com/2842920/110843404-8c9abd80-82a8-11eb-96ac-16c5a63e1677.png)
